From 4d7104a6984205df21175a088228dc5223215f00 Mon Sep 17 00:00:00 2001
From: Andreas Jaeger <aj@suse.de>
Date: Sat, 13 Sep 2014 09:48:14 +0200
Subject: [PATCH] Stop using intersphinx

Remove intersphinx from the docs build as it triggers network calls that
occasionally fail, and we don't really use intersphinx (links other
sphinx documents out on the internet)

This also removes the requirement for internet access during docs build.

This can cause docs jobs to fail if the project errors out on
warnings.

Change-Id: I71e941e2a639641a662a163c682eb86d51de42fb
Related-Bug: #1368910
---
 doc/source/conf.py | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 61798e9..3fa01c3 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -27,7 +27,7 @@ sys.path.extend([os.path.abspath('../../swift'), os.path.abspath('../..'),
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx',
+extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.pngmath',
               'sphinx.ext.ifconfig', 'oslo.sphinx']
 todo_include_todos = True
@@ -214,8 +214,3 @@ latex_documents = [
 
 # If false, no module index is generated.
 #latex_use_modindex = True
-
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'python': ('http://docs.python.org/', None),
-                       'nova': ('http://nova.openstack.org', None),
-                       'glance': ('http://glance.openstack.org', None)}
